.github/workflows/auto_assign.yml

```diff
@@ -1,2 +0,0 @@
-# used by ./workflows/pr-auto-assign.yml
-addAssignees: author
```

Comment:
why did you remove this?

Reply:
It was duplicated from .github/auto_assign.yml and Github showed errors.

In the env0 repo, this file sits in .github/ and not in ...workflows